lib/remote/apilistener-configsync.cpp

```diff
@@ -454,19 +457,47 @@ void ApiListener::SendRuntimeConfigObjects(const JsonRpcConnection::Ptr& aclient
 	Log(LogInformation, "ApiListener")
 		<< "Syncing runtime objects to endpoint '" << endpoint->GetName() << "'.";
 
-	for (const Type::Ptr& type : Type::GetAllTypes()) {
-		auto *dtype = dynamic_cast<ConfigType *>(type.get());
+	std::set<ConfigObject*> syncedObjects;
```

Comment:
I'd go for `std::unordered_set` here. It's operations are faster and in my totally unscientific local test[^1], it seemed to use a little less memory than `std::set`. Note that compared to #10148 (which sorts a hand full of objects once), this choice is much more relevant here (sorts many thousand of objects for each connection).

[^1]: I quickly inserted 2 million values into a `std::(unordered_)set<void*>` and checked the memory use with `time -v` (GNU time): `std::set` used around 97MB, `std::unordered_set` around 89MB.